google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/ChangeStreamRecord.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.models;
+
+import com.google.api.core.InternalExtensionOnly;
+
+/**
+ * Default representation of a change stream record, which can be a Heartbeat, a CloseStream, or a
+ * logical mutation.
+ */
+@InternalExtensionOnly("Intended for use by the BigtableIO in apache/beam only.")
+public interface ChangeStreamRecord {}
```

Comment:
does this not bleed to the surface in BigtableIO?

Reply:
No actually. Our beam code receives ChangeStreamRecord but will handle Heartbeat/CloseStream and return ChangeStreamMutation only to the BigrableIO.readChangeStream()